docs/platforms/unity/metrics/index.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Set Up Metrics
+description: "Learn how to measure the data points you care about by configuring Metrics in your Unity project."
+sidebar_order: 9000
+---
+
+<Include name="feature-stage-alpha-metrics.mdx" />
+
+<Note>
+
+Metrics for Unity are supported with Sentry Unity SDK version `2.0.0` and above.
+
+</Note>
+
+Sentry metrics help you pinpoint and solve issues that impact user experience and app performance by measuring the data points that are important to you. You can track things like processing time, event size, user signups, and conversion rates, then correlate them back to tracing data in order to get deeper insights and solve issues faster.
+
+## Initialization
+
+To enable metrics you have to opt-in the metrics feature.
```

Comment:
```suggestion
To enable metrics, opt in to the metrics feature:
```
It's "opt in" (two words) when used as a verb and "opt-in" (hyphenated) when used as an adjective (for example, the opt-in button)

Reply:
TIL! Thanks for the explanation!